src/main/java/com/iemr/mmu/service/dataSyncActivity/DownloadDataFromServerImpl.java

```diff
@@ -192,32 +159,25 @@ private int downloadDataFromServer(SyncDownloadMaster syncDownloadMaster, String
 		if (response != null && response.hasBody()) {
 			JSONObject obj = new JSONObject(response.getBody());
 			if (obj != null && obj.has("data") && obj.has("statusCode") && obj.getInt("statusCode") == 200) {
-				// JSONArray dataList = obj.getJSONArray("data");
-				// Consume the response from API in master data digester
 				MasterDownloadDataDigester masterDownloadDataDigester = InputMapper.gson().fromJson(response.getBody(),
 						MasterDownloadDataDigester.class);
 				if (masterDownloadDataDigester != null) {
 					int i = updateMastersWithLatestData(masterDownloadDataDigester, syncDownloadMaster);
 					if (i > 0)
 						successFlag = 1;
 				} else {
-					// error in parsing response data
```

Comment:
@Anjana2310 @swetatech24 In this class you have removed some valid comments too.